conf/application.conf

```diff
@@ -60,30 +60,35 @@ play.filters.headers.contentSecurityPolicy= "default-src 'self' 'unsafe-inline'
 # play.crypto.secret="5QbfH5vWzml20EJcocXTcwnG7QaXlfo1Tv2yXT6Bs2ZsuHnR0HchVm6oxiWOX9vG"
 
 microservice {
-    metrics {
-        graphite {
-            host = graphite
-            port = 2003
-            prefix = play.${appName}.
-            enabled = false
-        }
+  metrics {
+    graphite {
+      host = graphite
+      port = 2003
+      prefix = play.${appName}.
+      enabled = false
     }
+  }
 
-    services {
-        binding-tariff-classification {
-            host = localhost
-            port = 9580
-        }
+  services {
+    binding-tariff-classification {
+      host = localhost
+      port = 9580
     }
+
+    auth {
```

Comment:
is it needed?

Reply:
Yes, otherwise it wont work

Reply:
You would need to add this connection also in `app-config-base`